core/src/fwd/Kokkos_Fwd_HIP.hpp

```diff
@@ -50,7 +50,10 @@ namespace Kokkos {
 namespace Experimental {
 class HIPSpace;            ///< Memory space on HIP GPU
 class HIPHostPinnedSpace;  ///< Memory space on Host accessible to HIP GPU
-class HIP;                 ///< Execution space for HIP GPU
+#if defined KOKKOS_ENABLE_HIP_UNIFIED_MEMORY
+class HIPUnifiedSpace;  ///< Memory space on Host accessible to HIP GPU
```

Comment:
IMHO, this should be different from the description for `HIPHostPinnedSpace`.
```suggestion
class HIPUnifiedSpace;  ///< Memory migratable between Host and HIP GPU
```